docs/tables/circleci_build.md

```diff
@@ -0,0 +1,56 @@
+# Table: circleci_build
+
+Result of a single execution of a workflow.
```

Comment:
I think we could use the description provided in the table definition file.

`A CircleCI build is a result of a single execution of a workflow.`